rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -89,6 +90,28 @@ def __len__(self) -> int:
         """Returns the number of items currently stored in this buffer."""
         return len(self._storage)
 
+    @ExperimentalAPI
+    def add_batch(self, batch: SampleBatchType, **kwargs) -> None:
+        """Deprecated in favor of new ReplayBuffer API."""
```

Comment:
Could we use the `@Deprecated` decorator here (ray.rllib.utils.deprecation)?
Just put it above the method (new=..., old=..., error=False). It'll take care of the message (log_once) and everything else.